### DISCUSSION TITLE
Where's the guidebook on Neoforge?

### DISCUSSION THREAD

**Discussion by mosharky** [Rearth/Oritech#171] [Q&A]
From what I can see, Lavender is used on Fabric, but it doesn't exist on Neoforge. Is the guidebook unavailable for now?

**Comment by Rearth** [comment]
Hi, the guide book has been moved to an online wiki: https://moddedmc.org/en/mod/oritech/docs/getting_started/lasers as lavender isnt available on neoforge

**Reply by mosharky** [reply]
What about the multiblocks? I can't tell what blocks are needed to form them

**Reply by Rearth** [reply]
It ays you need machine cores when you interact with it. So you need machine cores. There's not much more to say here
